MANIFEST.in

```diff
@@ -2,7 +2,7 @@ recursive-include weldx/asdf/schemas *.yaml
 
 prune .binder
 prune .github
-prune conda.recipe
+prune devtools
```

Comment:
I take it we dont want to distribute those @marscher ?

Reply:
well a source dist could contain them, because it can be used to build the package. But I do not have a strong opinion about it. It does not waste so much memory

Reply:
right, well we can include it again later